### PR TITLE
Better setting of dialog query

### PR DESCRIPTION
**Pull request by stmeyer**
sets `dialog_action` query if the url is an object

### CODE REVIEW

File: webroot/js/lib/dialog.js [hunk 1]
```diff
@@ -259,6 +259,18 @@ Frontend.Dialog = Class.extend({
      * @return  mixed
      */
     _ensureDialogAction: function(url) {
+        if (typeof url === 'object') {
+            if (url.hasOwnProperty('query')) {
+                url.query.dialog_action = 1;
```
Comment (jonathan-neugber) [inline webroot/js/lib/dialog.js]: This is a really good idea. 

But a dialog action also requires `json_action` to be set to `1` :)

Comment (stmeyer) [inline webroot/js/lib/dialog.js]: `json_action` is set https://github.com/scherersoftware/cake-frontend-bridge/blob/master/webroot/js/lib/app.js#L115. But if you want, I'll set it in this place, too.

Comment (jonathan-neugber) [inline webroot/js/lib/dialog.js]: Oh yeah you are right.

**Comment by felixkempf** [comment]
Thanks @stmeyer 👍

**Pull request merged**